voice/methods/conferences/getConferences.md

```diff
@@ -0,0 +1,123 @@
+{% method %}
+
+## Retrieve Account's Conferences
+Returns a max of 1000 conferences, sorted by createdTime from oldest to newest
+
+### Request URL
+
+<code class="get">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter      | Description                                                                                                                      | Mandatory |
+|:---------------|:---------------------------------------------------------------------------------------------------------------------------------------------|
+| pageSize       | Specifies the max number of conferences that will be returned. Range: integer numbers from 1 to 1000. Default value is 1000.     | No        |
+| name           | Filter results by the `name` field.                                                                                              | No        |
+| minCreatedTime | Filter results to conferences which have a `createdTime` after or including `minCreatedTime` (in ISO8601 format).                | No        |
+| maxCreatedTime | Filter results to conferences which have a `createdTime` before or including `maxCreatedTime` (in ISO8601 format).               | No        |
+
+### Response Attributes
+
+| Property              | Description                                                                                          |
+|:----------------------|:-----------------------------------------------------------------------------------------------------|
+| id                    | The conference id.                                                                                   |
+| name                  | The conference name.                                                                                 |
+| createdTime           | The time the conference was initiated, in ISO 8601 format.                                           |
+| completedTime         | The time the conference was terminated, in ISO 8601 format.                                          |
```

Comment:
Worth adding `(optional)` here, to make it clear that ongoing conferences are also included?
(ongoing conferences are also included right?)

Reply:
Right